index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div class="head-content">
+            <p class="logo">
+               <a href="http//">Welcome</a>
+            </p>
+            <p class="nav">
+                <span></span>
+                <span></span>
+                <span></span>
+            </p>
+        </div>
+    </header>
+    <section class="presentation">
+    <div class="content">
+            <h1> Hey There, I'm Ghislain </h1>
+            <h1 class="intros">I'm a software developer</h1>
```

Comment:
- [ OPTIONAL] We need to have only one `<h1>` element in an HTML page. The reason for that is for semantic reasons such as SEO and accessibility of the page by screen readers. Therefore, kindly use other heading tags  from `<h2> ... <h6>`